WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -118,6 +113,32 @@ public final class WordPressComOAuthClient: NSObject {
         self.wordPressComApiBaseUrl = wordPressComApiBaseUrl
     }
 
+    /// Authenticates on WordPress.com using the OAuth endpoints.
+    ///
+    /// Delegates to `authenticateWithUsername(_:, password:, multifactorCode:, needsMultifactor:, success:, failure:)`.
+    /// Here to provide source-compatibility with the Objective-C layer, where using a default parameter (`needsMultifactor: ... = nil`) doesn't result in a method signature without the parameter.
+    ///
+    /// - Parameters:
+    ///     - username: the account's username.
+    ///     - password: the account's password.
+    ///     - multifactorCode: Multifactor Authentication One-Time-Password. If not needed, can be nil
+    ///     - success: block to be called if authentication was successful. The OAuth2 token is passed as a parameter.
+    ///     - failure: block to be called if authentication failed. The error object is passed as a parameter.
+    @objc public func authenticateWithUsername(_ username: String,
```

Comment:
Do you think we should annotate this function as "to be deprecated in next versions"?

Reply:
Good idea. I'll do it as part of shipping a new release.